### PR TITLE
Enable verifier fuzzer in github CI/CD

### PR DESCRIPTION
**Pull request by dthaler**
Signed-off-by: Dave Thaler <dthaler@microsoft.com>

## Description

This is another piece split out from PR #1153

## Testing

This PR adds tests to ci/cd

## Documentation

Already documented

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -152,6 +152,20 @@ jobs:
...
 
+  # Run the verifier fuzzer.
+  verifier_fuzzer:
+    needs: libfuzzer
+    # Always run this job.
+    if: github.event_name == 'schedule' || github.event_name == 'pull_request'
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      name: verifier_fuzzer
+      test_command: verifier_fuzzer.exe verifier_corpus -use_value_profile=1 -max_total_time=1800
```
Comment (Alan-Jowett) [inline .github/workflows/cicd.yml:163]: Once the #1164 is merged add "-artifact_prefix=Artifacts\" to the test_command so that artifacts are uploaded.

Comment (Alan-Jowett) [inline .github/workflows/cicd.yml:163]: Once the #1164 is merged add "-artifact_prefix=Artifacts\" to the test_command so that artifacts are uploaded.

**Review by Alan-Jowett — DISMISSED** [review]

**Comment by Alan-Jowett** [comment]
bpf2c fuzzing failure should be fixed by https://github.com/microsoft/ebpf-for-windows/pull/1189

File: .github/workflows/cicd.yml (hunk 1 above)
Comment (dthaler) [reply .github/workflows/cicd.yml]: Done

**Review by Alan-Jowett — DISMISSED** [review]

**Review by Alan-Jowett — APPROVED** [review]

**Comment by Alan-Jowett** [comment]
bpf2c fuzzer is asserting on checked builds due to #1212

**Pull request merged**